lib/display.ex

```diff
@@ -53,6 +53,12 @@ defmodule Display do
     format_error(error) |> IO.puts
   end
 
+  def congratulate do
+    format_green("\nYou have learned much. You must find your own path now.")
+    |> IO.puts
+    exit(:normal)
```

Comment:
Yeah, I thought so too, but it [wasn't unprecedented](https://github.com/elixirkoans/elixir-koans/blob/master/lib/display.ex#L10). Either way, I'll update this PR.